wled00/wled.cpp

```diff
@@ -752,7 +752,22 @@ void WLED::initConnection()
     // convert the "serverDescription" into a valid DNS hostname (alphanumeric)
     char hostname[25];
     prepareHostname(hostname);
-    WiFi.begin(multiWiFi[selectedWiFi].clientSSID, multiWiFi[selectedWiFi].clientPass);
+    int16_t nets = WiFi.scanNetworks();
```

Comment:
The unfortunate problem with this call is that it is a blocking call.
While it is scanning LEDs are not updated.

Reply:
I made the scanNetwork() async in [5cd582cf](https://github.com/Aircoookie/WLED/pull/3716/commits/5cd582cfd16c9da9c6a984154d5c8fddff300dd3)